test/model.discriminator.querying.test.js

```diff
@@ -30,15 +30,15 @@ const ImpressionEventSchema = new BaseSchema({ element: String });
 const ConversionEventSchema = new BaseSchema({ revenue: Number });
 const SecretEventSchema = new BaseSchema({ secret: { type: String, select: false } });
 
-describe('model', function() {
-  describe('discriminator()', function() {
+describe('model', function () {
```

Comment:
Can you please undo these formatting changes? They're likely to cause merge conflicts for us, and they make this PR more difficult to review.

Reply:
My bad, fixed